src/main/java/bio/terra/cli/command/Status.java

```diff
@@ -22,13 +23,18 @@ protected void execute() {
     formatOption.printReturnValue(statusReturnValue, this::printText);
   }
 
+  @SuppressFBWarnings(
+      value = {"URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"},
+      justification = "This POJO class is used for easy serialization to JSON using Jackson.")
   /** POJO class for printing out this command's output. */
```

Comment:
Yes. These classes define the output JSON format for a command. Now that we have tests calling the commands, I made these classes public so the tests can re-serialize them for easier assertions.

Reply:
If it's just for serialization, you might also consider annotating the class with `@JsonAutoDetect` (per 4.1 [here](http://www.javabyexamples.com/jackson-can-serialize-private-fields)), it might make the intent clearer without having to make everything visible.

Reply:
Hmm just tried that and it doesn't seem to work. I think that's because the tests also read the properties of the serialized object in order to do assertions.
e.g. `assertTrue(authStatus.loggedIn, "auth status indicates user is logged in");`

Reply:
Thanks for looking into it, sorry it was a dead end!

Reply:
No, that's good to know about anyway, thanks!